sisl/io/xsf.py

```diff
@@ -391,6 +484,62 @@ def _setup(self, *args, **kwargs):
         if not hasattr(self, '_md_steps'):
             self._md_steps = 1
 
+    def read_geometry(self, data=False, step=0):
+        """ Returns Geometry object from the given step of the AXSF file
+
+        Parameters
+        ----------
+        data : bool, optional
+           in case the XSF file has auxiliary data, return that as well.
+        step : int, default 0
+           the step/snapshot index (0-indexed) to load.
+           Please see ``read_geometries`` if you need to load several.
+        """
+        if step < 0:
+            step = self.read_num_steps() + step
+        return self._read_geom_many_step([step], load_data=data, squeeze=True)
+
+    def read_geometries(self, steps=None, one_geometry=False, load_data=False):
```

Comment:
A small comment, in other `read_geometry` variants, I allow keywords to control what to return, see e.g. `outSileSiesta.read_geometry`, I would prefer one such method.

Reply:
Hmm okay you would like read_geometries to be the same method as read_geometry?

Reply:
That would make good sense actually. Ill do that.

Reply:
Yes :)
Then arguments determine whether 1 or more geometries will be returned.